docs/advanced-topics/dockstore-cli/local-cromwell-config.rst

```diff
@@ -0,0 +1,37 @@
+Setting Up Cromwell for Local Runs
+==================================
+
+Cromwell, when running locally, loses the ability to set compute resources within what is specified in a task's runtime attributes. This is a problem with scattered tasks, as by default, every instance of a scattered task may attempt to execute at the same time. As such, the local version of Cromwell sometimes uses too many resources when running scattered tasks, causing some or all instances of the scattered task to get [sigkilled](https://www.gnu.org/software/libc/manual/html_node/Termination-Signals.html). There is also the possability of concurrent instances of a scattered task locking up Docker for your operating system. 
+
+.. tip::  If a Docker lockup happens, you will notice tasks do not progress beyond WaitingForReturnCode and you will be temporarily unable to use Docker on your OS. This can be resolved by restarting Docker on your machine.
+
+The easiest way to avoid these issues is to set up a Cromwell configuration file that sets concurrent-job-limit. Of course, there is a tradeoff: If you set a concurrent-job-limit, tests involving scattered tasks will execute slower as less instances of a scattered task will run in parallel. That being said, we still recommend setting this value when running on a local machine, as it makes Cromwell much more stable.
+
+1. Download `this template file <https://github.com/broadinstitute/cromwell/blob/develop/cromwell.example.backends/cromwell.examples.conf>`__ and name it ``.cromwell.conf``
```

Comment:
Where does `.cromwell.conf` go? I don't know, but I'm sure it has to be somewhere specific.

Reply:
Usually the User folder, although I don't think it actually needs to be somewhere specific provided you give the full path in the Dockstore config file... which I forgot to copy over. Added in the next commit.